lib/jekyll/utils.rb

```diff
@@ -203,7 +203,10 @@ def slugify(string, mode: nil, cased: false)
       end
 
       # Drop accent marks from latin characters. Everything else turns to ?
-      string = ::I18n.transliterate(string) if mode == "latin"
+      if mode == "latin"
+        I18n.config.available_locales = :en if I18n.config.available_locales.empty?
```

Comment:
Should we set `available_locales` to an array (`[:en]`) instead?

Reply:
I don't think its necessary.
If you were to add a `print I18n.config.available_locales` immediately after this line, the output would return `[:en]`

Reply:
👍 Thanks for checking!

Reply:
This code only gets run if `mode` is set to `latin`. If it's not you break everyone else's code by removing `I18n.config.available_locales = :en` from jekyll module which is exactly what happened to me.

Reply:
@milosgajdos83 `I18n.config.available_locales = :en` was specifically added to the `Jekyll` module for the sole use in the `slugify` filter.
Please open a *new* separate issue for better tracking..